test/performance-split-words-time.ts

```diff
@@ -0,0 +1,39 @@
+import test, { ExecutionContext } from "ava";
+const start = process.hrtime();
+console.log("PERFORMANCE TEST: split words time");
+console.log("Importing split function (which imports WordBreakProperty.ts)");
+import { split } from "../lib";
+const end = process.hrtime(start);
+console.log(`Importing time: ${end[0]}s ${end[1] / 1000000}ms`);
+import { readFileSync } from "fs";
+
+console.log("Importing book");
+
+let book = readFileSync("./test/book-example.txt", "utf-8");
```

Comment:
It's a bit ridiculous to include such a large sample file in the repo. Especially, since it works strangely in diffs. I would either add a `.gitattributes` file that marks this as non-text, like so:

```ini
test/book-example.txt -text
```

Or store it in the repo in a compressed format, such as a gzip.